sections/shared/modals/SelectTokenModal/SelectTokenModal.tsx

```diff
@@ -32,6 +31,8 @@ import Connector from 'containers/Connector';
 import { RowsHeader, RowsContainer, CenteredModal } from '../common';
 
 import TokenRow from './TokenRow';
+import useSynthetixQueries from '@synthetixio/queries';
+import _ from 'lodash';
```

Comment:
it's probably better to require the functions you need right? Otherwise webpack would import the whole library

Reply:
done